File: src/main/java/com/example/demo/Doc.java
package com.example.demo;

public class Doc {
    public String URL;
    public String Title;
    public int Rank = 0;
    public double TF = 0;
    public String Word;

    public Doc(String URL, String title, int rank, double TF, String word) {
        this.URL = URL;
        this.Title = title;
        this.Rank = rank;
        this.TF = TF;
        this.Word = word;
    }

    public Doc() {

    }
}

File: ui/src/Doc.js
import React from 'react';

const Doc = ({ doc }) => {
  return (
    <div className="doc-container">
      <a href={doc["Url"]} target="_blank" rel="noopener noreferrer"><h3>{doc["Title"]}</h3></a>
      {doc["Url"]}
      <div className='lolo'><span dangerouslySetInnerHTML={{ __html: doc["Describtion"] }}></span>
      </div>
      
    </div>
  );
};

export default Doc;
